components/travel.js

```diff
@@ -0,0 +1,272 @@
+const Travel = () => {
+  return (
+    <>
+      {" "}
+      <section className="home-section" id="travel">
+        <div className="container">
+          <div className="row pt-5">
+            <div className="col-md-12 pt-3 pb-5">
+              <h2
+                h2
```

Comment:
Please align the heading in centre as it's done for other pages.
1. Blog: https://in.pycon.org/blog/
2. FAQ: https://in.pycon.org/2023/faq/

Reply:
Centre aligned the heading, added the same styles to FAQ and COC to maintain a theme